From ac9fd659ef8cac9a71f0ef943ef6da04f1939775 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Sun, 11 Aug 2019 16:37:25 +0200
Subject: drm: gm12u320: Add -ENODEV to list of errors to ignore
Git-commit: ac9fd659ef8cac9a71f0ef943ef6da04f1939775
Patch-mainline: v5.4-rc1
References: bsc#1152472

Add -ENODEV to the list of usb-transfer errors which we ignore to
avoid logging Frame update errors when the device gets unplugged.

Acked-by: Sam Ravnborg <sam@ravnborg.org>
Signed-off-by: Hans de Goede <hdegoede@redhat.com>
Link: https://patchwork.freedesktop.org/patch/msgid/20190811143725.5951-2-hdegoede@redhat.com
Acked-by: Thomas Zimmermann <tzimmermann@suse.de>
---
 drivers/gpu/drm/tiny/gm12u320.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/tiny/gm12u320.c b/drivers/gpu/drm/tiny/gm12u320.c
index aa4b46e47e19..03d0e2df6774 100644
--- a/drivers/gpu/drm/tiny/gm12u320.c
+++ b/drivers/gpu/drm/tiny/gm12u320.c
@@ -422,7 +422,7 @@ static void gm12u320_fb_update_work(struct work_struct *work)
 	return;
 err:
 	/* Do not log errors caused by module unload or device unplug */
-	if (ret != -ECONNRESET && ret != -ESHUTDOWN)
+	if (ret != -ENODEV && ret != -ECONNRESET && ret != -ESHUTDOWN)
 		GM12U320_ERR("Frame update error: %d\n", ret);
 }
 
-- 
2.28.0

